app/src/main/java/ml/docilealligator/infinityforreddit/fragments/ViewRedditGalleryImageOrGifFragment.java

```diff
@@ -272,6 +273,7 @@ public void onFail(Exception error) {
 
             if (!activity.isUseBottomAppBar()) {
                 bottomAppBarMenu.setVisibility(View.GONE);
+                bottomAppBar.setVisibility(View.VISIBLE);
```

Comment:
Why is the visibility View.VISIBLE here?

Reply:
I set this appbar visible so images with captions will always be on the screen but empty captions will disappear, since I think that was how captions were previously done. Altho testing this, I think i would prefer if the captions disappear/show when the user taps on the image. 

I'll close this pr for now since I haven't had time recently, and circle back to this when my schedule is free! 🙂